specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

```diff
@@ -4312,6 +4312,18 @@
       },
       "description": "The instance view of a resource."
     },
+    "AvailabilitySetSkuType": {
```

Comment:
I don't see this referenced anywhere, is that expected?

Reply:
Yes, the main idea is to provide a set of constant to the users to specify possible values for skuName while creating Availability Set.

Reply:
Unfortunately we had used the same SKU object at different places and it does not mean the same when used in context of an AvailabilitySet as compared to VMScaleSet or in GetSKUs APIs. Hence this is just to help customers to tell them what are the possible values when creating an Availability Set.

Reply:
So these values go in the Sku.Name field?  I don't think adding a new type that isn't referenced is the best way to solve this, isn't updating the description sufficient?

Reply:
No it is not. We don't want users to guess the values and type anything random in there. Do you any side-effect of having it? It seems more helpful than just have comments.

Reply:
Sorry I should have rephrased that, since this type isn't referenced the code generators throw it away.  If you look at some of the generated PRs you'll see that `AvailabilitySetSkuType` isn't added to the models.  So since you can't update the models the best you can do is update the docs.

Reply:
I had checked the generated C# code. It shows up as a static class with 2 string variables there.

Reply:
Interesting, it doesn't appear for other languages.  @fearthecowboy can you comment on this?

Reply:
Ugh nevermind I was looking at the wrong SDK PR.